File: src/main/java/com/gitlab/daring/image/command/env/SetTaskCommand.java
package com.gitlab.daring.image.command.env;

import com.gitlab.daring.image.command.BaseCommand;
import com.gitlab.daring.image.command.CommandEnv;

public class SetTaskCommand extends BaseCommand {

	public SetTaskCommand(String... args) {
		super(args);
	}

	@Override
	public boolean isEnabled(CommandEnv env) {
		return true;
	}

	@Override
	public void execute(CommandEnv env) {
		env.curTask = arg(0, "");
	}

}

File: src/main/kotlin/com/gitlab/daring/image/command/env/SetTaskCommand.kt
package com.gitlab.daring.image.command.env

import com.gitlab.daring.image.command.CommandEnv
import com.gitlab.daring.image.command.KBaseCommand

internal class SetTaskCommand(vararg args: String) : KBaseCommand(*args) {

    override fun isEnabled(env: CommandEnv) = true

    override fun execute(env: CommandEnv) {
        env.curTask = arg(0, "")
    }

}